src/ogma/features/nodeGrouping.ts

```diff
@@ -305,23 +312,27 @@ export class NodeGroupingTransformation {
    * Run the circle pack layout on the subnodes
    * @param subNodes
    */
-  private async _runCirclePack(subNodes: NodeList<LkNodeData, LkEdgeData>): Promise<void> {
-    await this._ogma.algorithms.circlePack({
-      nodes: subNodes,
-      margin: 10,
-      sort: 'asc'
-    });
+  private _runCirclePack(subNodes: NodeList<LkNodeData, LkEdgeData>): Promise<Point[]> {
+    return Promise.resolve(
+      this._ogma.algorithms.circlePack({
+        nodes: subNodes,
+        margin: 10,
+        sort: 'asc',
+        dryRun: true
+      })
+    );
   }
 
   private async _runChainLayout(subNodes: NodeList<LkNodeData, LkEdgeData>): Promise<void> {
```

Comment:
can you add comments to this method. This is complex to understand (every line basically)